apps/sms/test/marionette/drafts_test.js

```diff
@@ -283,6 +289,34 @@ marionette('Messages Drafts', function() {
       });
     });
 
+    suite('Thread-less recipient-less drafts', function() {
```

Comment:
I put this test in its own suite. Didn't find any easy way to include that in `Thread-less drafts` test suite... Do you have a better idea?